kvbc/include/replica_resources.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 VMware, all rights reserved
+
+#pragma once
+
+#include "ISystemResourceEntity.hpp"
+#include "IncomingMsgsStorageImp.hpp"
+
+class ReplicaResourceEntity : public concord::performance::ISystemResourceEntity {
```

Comment:
This ReplicaResourceEntity implementation is hijacking the responsibility of IResourceManager. Implementation of that class should take in a constructor all necessary measurements providers (IResourceEntity) and operate with them the way creator intents to. If needs e.g. distincted NetworkMonitorResourceEntity class A : public IResourseManager { A(IResourceEntity* networkMonitorResourceEntity , ... ) }

Reply:
Sorry, I don't understand this sentence. 
As we discussed, I see it as a raw source for measurements that the user e.g. ResourceMansger (the strategy) can call and use as it likes